setup.py

```diff
@@ -18,12 +19,15 @@
 
     url='https://github.com/jeancochrane/pytest-flask-sqlalchemy-transactions',
     description='Run tests in transactions using pytest, Flask, and SQLalchemy.',
-    long_description=__doc__,
+    long_description=readme(),
     license='MIT',
 
     packages=['transactions'],
-    install_requires=requirements,
-
+    install_requires=['pytest>=3.2.1',
+                      'pytest-mock>=1.6.2',
+                      'SQLAlchemy>=1.2.2',
+                      'Flask-SQLAlchemy>=2.3'],
```

Comment:
Are these the minimum versions of the dependencies this library requires?

Reply:
That's a great question -- I'm not entirely sure. Is there a principled approach to testing this, or should I do some trial and error?

Reply:
I don't know of a principled approach.